MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include README.md
-include LICENSE.txt
```

Comment:
Maybe the LICENCE should be retained

Reply:
LICENSE is added through `pyproject.toml` [here](https://github.com/confluentinc/confluent-kafka-python/blob/dev_pyproject-toml/pyproject.toml#L18)